datacite-draft-doi-handler/src/test/java/no/unit/nva/doi/DraftDoiHandlerTest.java

```diff
@@ -0,0 +1,110 @@
+package no.unit.nva.doi;
+
+import static no.unit.nva.hamcrest.DoesNotHaveNullOrEmptyFields.doesNotHaveNullOrEmptyFields;
+import static nva.commons.utils.attempt.Try.attempt;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.nio.file.Path;
+import no.unit.nva.doi.model.DoiUpdateDto;
+import no.unit.nva.events.models.AwsEventBridgeDetail;
+import no.unit.nva.events.models.AwsEventBridgeEvent;
+import no.unit.nva.publication.doi.dto.Publication;
+import no.unit.nva.publication.doi.dto.PublicationDtoTestDataGenerator;
+import no.unit.nva.publication.doi.dto.PublicationHolder;
+import nva.commons.utils.IoUtils;
+import nva.commons.utils.JsonUtils;
+import org.hamcrest.MatcherAssert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class DraftDoiHandlerTest {
+
+    public static final String DOI_PUBLICATION = "doi.publication";
+    public static final String ERROR_MESSAGE = "error";
+
+    private DraftDoiHandler handler;
+    private ByteArrayOutputStream outputStream;
+    private Context context;
+    private AwsEventBridgeEvent<AwsEventBridgeDetail<PublicationHolder>> event;
+
+    @BeforeEach
+    public void setUp() {
+        handler = new DraftDoiHandler();
+        outputStream = new ByteArrayOutputStream();
+        context = mock(Context.class);
+        event = mock(AwsEventBridgeEvent.class);
+    }
+
+
+    @Test
+    public void handleRequestReturnsDoiUpdateDtoWithPublicationUriWhenInputIsValid() {
+        InputStream inputStream = IoUtils.inputStreamFromResources(
+            Path.of("doi_publication_event_valid.json"));
+        handler.handleRequest(inputStream,outputStream,context);
+        DoiUpdateDto response = parseResponse();
+        assertThat(response.getPublicationId(), is(not(nullValue())));
```

Comment:
Should also contain a ``modificationTimestamp` ? Separate test for it?|
To my knownledge the `modificationTimestamp` and `publicationId` is the required attributes in the `DoiStatusUpdate`.